marketplace_register.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+######## RHM Registration Script #############
+
+# create namespace
+oc create namespace redhat-marketplace
+
+# Create the markeplace subscription 
+oc apply -f "https://marketplace.redhat.com/provisioning/v1/rhm-operator/rhm-operator-subscription?approvalStrategy=Automatic"
+
+# Create Red Hat Marketplace Kubernetes Secret [change the pull secret in the command with yours  you got from RH Marketplace]
+oc create secret generic redhat-marketplace-pull-secret -n redhat-marketplace --from-literal=PULL_SECRET=<pullsecret>
```

Comment:
Maybe it's better to export the pull_secret as an environment variable, and then call it

Reply:
ok, will do. i didn't have time to go back on this PR